src/test/java/com/sprint/mission/discodeit/repository/MessageRepositoryTest.java

```diff
@@ -0,0 +1,80 @@
+package com.sprint.mission.discodeit.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.sprint.mission.discodeit.config.QuerydslConfig;
+import com.sprint.mission.discodeit.entity.message.Message;
+import com.sprint.mission.discodeit.logging.LogConfiguration;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.context.annotation.Import;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Slice;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.domain.Sort.Direction;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@DataJpaTest
+@ActiveProfiles("test")
+@Import({LogConfiguration.class, QuerydslConfig.class})
+@ExtendWith(SpringExtension.class)
+class MessageRepositoryTest {
```

Comment:
```suggestion
@DataJpaTest
@ActiveProfiles("test")
@Import({LogConfiguration.class, QuerydslConfig.class})
class MessageRepositoryTest {
```

DataJpaTest 내부에 이미 ExtendWith(SpringExtension.class)가 포함되어 있습니다!